src/comfystream/scripts/build_trt.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import time
+import argparse
+
+# Reccomended running from comfystream conda environment
+# in devcontainer from the workspace/ directory, or comfystream/ if you've checked out the repo
+# $> conda activate comfystream
+# $> python src/comfystream/scripts/build_trt.py --model /ComfyUI/models/checkpoints/SD1.5/dreamshaper-8.safetensors --out-engine /ComfyUI/output/tensorrt/static-dreamshaper8_SD15_$stat-b-1-h-512-w-512_00001_.engine
+
+# Paths path explicitly to use the downloaded comfyUI installation on root
+ROOT_DIR="/"
+COMFYUI_DIR = "/ComfyUI"
+
```

Comment:
This will resolve the runtime error, however I noticed that cache files are created at the root from where the script is ran at `cuda_temp/`, is this what you're looking to solve with this cache path?

```suggestion
COMFYUI_DIR = "/ComfyUI"
timing_cache_path = "/ComfyUI/output/tensorrt/timing_cache"
```